<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>998</width>
    <height>670</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>998</width>
    <height>670</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>4</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string extracomment="Helmet Overalls Detection Demo">鲲云星空加速卡解决方案</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#MainWindow{background-color: rgb(13, 79, 114);}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>998</width>
     <height>670</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1920</width>
     <height>1080</height>
    </size>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout" stretch="4,15">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>10</number>
    </property>
    <property name="topMargin">
     <number>10</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <property name="bottomMargin">
     <number>10</number>
    </property>
    <item>
     <widget class="QWidget" name="widget_1" native="true">
      <property name="styleSheet">
       <string notr="true">#widget_1{background-color: rgb(13, 79, 114);}
</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0" colspan="5">
        <widget class="QLabel" name="label_4">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>60</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>60</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Source Code Pro</family>
           <pointsize>15</pointsize>
           <weight>75</weight>
           <italic>false</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);
border-image: url(:/new/prefix1/images/title_0.png);</string>
         </property>
         <property name="text">
          <string>鲲云智慧安监解决方案</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignHCenter|Qt::AlignTop</set>
         </property>
         <property name="margin">
          <number>5</number>
         </property>
         <property name="indent">
          <number>-1</number>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="DisplayVideoButton">
           <property name="maximumSize">
            <size>
             <width>58</width>
             <height>24</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(13, 79, 114);

color: rgb(255, 255, 255);
</string>
           </property>
           <property name="text">
            <string>视频</string>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="DisplayCameraButton">
           <property name="maximumSize">
            <size>
             <width>58</width>
             <height>24</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(13, 79, 114);
color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string>摄像头</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="DisplayLocalTestButton">
           <property name="maximumSize">
            <size>
             <width>58</width>
             <height>24</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(13, 79, 114);
color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string>图片</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="DisplayLocalNextButton">
           <property name="maximumSize">
            <size>
             <width>58</width>
             <height>24</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(13, 79, 114);
color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string>下一张</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="startGetPoint">
           <property name="maximumSize">
            <size>
             <width>58</width>
             <height>24</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(13, 95, 110);
color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string>开始画框</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="stopGetPoint">
           <property name="maximumSize">
            <size>
             <width>58</width>
             <height>24</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(13, 95, 110);
color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string>结束画框</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="clearROI">
           <property name="maximumSize">
            <size>
             <width>58</width>
             <height>24</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(13, 95, 110);
color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string>清除区域</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <widget class="QCheckBox" name="retSave">
             <property name="maximumSize">
              <size>
               <width>25</width>
               <height>19</height>
              </size>
             </property>
             <property name="layoutDirection">
              <enum>Qt::LeftToRight</enum>
             </property>
             <property name="autoFillBackground">
              <bool>false</bool>
             </property>
             <property name="styleSheet">
              <string notr="true">QCheckBox::indicator {width: 19px; height: 19px}</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_5">
             <property name="maximumSize">
              <size>
               <width>40</width>
               <height>49</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(255, 255, 255);</string>
             </property>
             <property name="text">
              <string>存图</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="label">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="4">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QComboBox" name="comboBox">
           <property name="maximumSize">
            <size>
             <width>102</width>
             <height>23</height>
            </size>
           </property>
           <property name="currentText">
            <string>未选择</string>
           </property>
           <item>
            <property name="text">
             <string>未选择</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>安全帽检测</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>红色工作服检测</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>蓝色工作服检测</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>人员入侵检测</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>皮肤裸露检测</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>烟火检测</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>反光衣检测</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>抽烟检测</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>打电话检测</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>漏油检测-室内</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>漏油检测-室外</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>人员倒地</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="resetCheck">
           <property name="layoutDirection">
            <enum>Qt::RightToLeft</enum>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="styleSheet">
            <string notr="true">QCheckBox::indicator {
width: 19px;
height: 19px;
};</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="resetCard">
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>24</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(13, 79, 114);
color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string>重置板卡</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="StartRun">
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>24</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(13, 79, 114);
color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string>运行</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="StopRun">
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>24</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(13, 79, 114);
color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string>停止</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_2" native="true">
      <property name="styleSheet">
       <string notr="true">#widget_2{border-image: url(:/new/prefix1/images/main-bg.png);background-color: rgb(13, 79, 114);}</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3" rowstretch="1,0" rowminimumheight="0,0">
       <property name="leftMargin">
        <number>15</number>
       </property>
       <property name="topMargin">
        <number>20</number>
       </property>
       <property name="rightMargin">
        <number>20</number>
       </property>
       <property name="bottomMargin">
        <number>17</number>
       </property>
       <property name="verticalSpacing">
        <number>2</number>
       </property>
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout_2">
         <property name="topMargin">
          <number>20</number>
         </property>
         <property name="verticalSpacing">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="VideoLabel" name="ResultLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>842</width>
             <height>428</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>1800</width>
             <height>900</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QProgressBar {border-radius: 5px;background: black;text-align: center;}
QProgressBar::chunk {border-radius:5px;border:1px solid black;background:#1ca1ca;}</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>14</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>14</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(175, 171, 171);</string>
           </property>
           <property name="text">
            <string>Version:AJ03-0307-333-2012E</string>
           </property>
           <property name="textFormat">
            <enum>Qt::PlainText</enum>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>14</width>
             <height>14</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QProgressBar" name="taskBar">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>180</width>
             <height>14</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>380</width>
             <height>14</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(2, 34, 50);
color: rgb(255, 255, 255);</string>
           </property>
           <property name="value">
            <number>0</number>
           </property>
           <property name="textVisible">
            <bool>false</bool>
           </property>
           <property name="invertedAppearance">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>750</width>
             <height>14</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>VideoLabel</class>
   <extends>QLabel</extends>
   <header location="global">videolabel.h</header>
   <slots>
    <slot>showImage(QImage)</slot>
    <slot>showText(QString)</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
